provider/sidecar_test.go

```diff
@@ -92,7 +93,37 @@ func TestSidecar(t *testing.T) {
 					ID:       "008",
 					Name:     "api",
 					Hostname: "another-aws-host",
+					Status:   0,
+					Ports: []service.Port{
+						service.Port{
```

Comment:
Nice catch @relistan! So, it turns out that the standard `goimports` [doesn't support](https://github.com/golang/go/issues/8759) the `-s` flag that `gofmt` provides. I ended up installing [this](https://github.com/jzelinskie/tools) 3rd party `goimports` to have this flag and keep using the `goimports` hook on save.